improver_tests/cli/test_init.py

```diff
@@ -200,6 +200,27 @@ def test_basic(self, m):
         self.assertEqual(m.call_count, 2)
 
 
+class Test_clizefy(unittest.TestCase):
+    """Test the clizefy decorator function"""
+
+    @patch('improver.cli.docutilize', return_value=None)
+    def test_called_once(self, m):
+        """Tests docutilize is called only once"""
+
+        def func():
+            """Dummy"""
+
+        clizefy(clizefy(func)).cli('argv[0]', '--help')
+        m.assert_called_once_with(func.__doc__)
+
+    @patch('improver.cli.partial')
+    def test_no_argument(self, m):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4888025)